src/sagas/walletConnect.js

```diff
@@ -45,6 +45,7 @@
  * loaded.
  */
 
+import '@walletconnect/react-native-compat';
```

Comment:
This adds a set of shims, which can be read here:

https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/packages/react-native-compat/index.js